runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -408,6 +412,12 @@ public void onChatMessage(ChatMessage event)
 
 			return;
 		}
+		
+		if (message.equals(HESPORI_LOOTED_MESSAGE))
```

Comment:
This message is not unique (it is used for trees, bushes, etc.) and it is possible to get loot from clearing tree patches in the form of tree roots. Before checking for message equality, check that the client's region is equal to the hespori region.

![2019-07-09_23-37-58](https://user-images.githubusercontent.com/2199511/60946353-88511c80-a2dd-11e9-81d9-eb7854cae12e.png)

Reply:
Okay I was pretty confused because I tested the trees and bushes etc, and the message did not come up. However, you are right, it comes up when you clear a dead tree (can you get loot from dead stuff,? Just out of interest). I'll make the necessary changes and check the client region.

Reply:
afaik only healthy normal trees (that is, not fruit or hardwood trees) give any loot when dug up.